{"content": {"suites": {"0": {"status": {"total_pass": 0, "total_skip": 0, "total_xpass": 0, "total_xfail": 0, "total_rerun": 0, "total_fail": 7, "total_error": 0}, "tests": {"0": {"status": "FAIL", "message": "   TypeError: Test_Login.test_Entry() got an unexpected keyword argument 'From'\n", "test_name": "test_Entry_1", "rerun": "0"}, "1": {"status": "FAIL", "message": "   TypeError: Test_Login.test_Entry() got an unexpected keyword argument 'From'\n", "test_name": "test_Entry_2", "rerun": "0"}, "2": {"status": "FAIL", "message": "   TypeError: Test_Login.test_Entry() got an unexpected keyword argument 'From'\n", "test_name": "test_Entry_3", "rerun": "0"}, "3": {"status": "FAIL", "message": "   TypeError: Test_Login.test_Exit() got an unexpected keyword argument 'From'\n", "test_name": "test_Exit_1", "rerun": "0"}, "4": {"status": "FAIL", "message": "   TypeError: Test_Login.test_Exit() got an unexpected keyword argument 'From'\n", "test_name": "test_Exit_2", "rerun": "0"}, "5": {"status": "FAIL", "message": "   TypeError: Test_Login.test_Exit() got an unexpected keyword argument 'From'\n", "test_name": "test_Exit_3", "rerun": "0"}, "6": {"status": "FAIL", "message": "   TypeError: Test_Login.test_Spotman() missing 1 required positional argument: 'setup'\n", "test_name": "test_Spotman", "rerun": "0"}}, "suite_name": "TestCases/test_login.py"}}}, "date": "October 21, 2022", "start_time": 1666336075.8463142, "total_suite": 1, "status": "FAIL", "status_list": {"pass": "0", "fail": "7", "skip": "0", "error": "0", "xpass": "0", "xfail": "0", "rerun": "0"}, "total_tests": "7"}